src/catalog/catalog.cpp

```diff
@@ -312,7 +200,7 @@ void CatalogContent::addScalarMacroFunction(
     macros.emplace(std::move(name), std::move(macro));
 }
 
-void CatalogContent::validateStorageVersion(storage_version_t savedStorageVersion) const {
+void CatalogContent::validateStorageVersion(storage_version_t savedStorageVersion) {
```

Comment:
I don't think this function changes any fields of catalogContent, why not make it const?

Reply:
This function is changed to static. I don't think it requires any fields of the class.